<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VFSAInversionDialog</class>
 <widget class="QDialog" name="VFSAInversionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>566</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configuración de la inversión VFSA</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="parametersGroupBox">
     <property name="title">
      <string>Parámetros del algoritmo</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="defaultParametersRadioButton">
          <property name="text">
           <string>Por defecto</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="customParametersRadioButton">
          <property name="text">
           <string>Personalizado</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="parametersFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="5">
          <widget class="QDoubleSpinBox" name="errorDoubleSpinBox">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>0.000100000000000</double>
           </property>
           <property name="maximum">
            <double>0.100000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>0.001000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="iterationsSpinBox">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="modelsPerTemperatureLabel">
           <property name="text">
            <string>Modelos por temperatura:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="iterationsPerSolutionLabel">
           <property name="text">
            <string>Iteraciones por solución:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QSpinBox" name="solutionsSpinBox">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="temperatureLabel">
           <property name="text">
            <string>Temperatura inicial:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="movesPerTemperatureSpinBox">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="errorLabel">
           <property name="text">
            <string>Error máximo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="temperatureDoubleSpinBox">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.001000000000000</double>
           </property>
           <property name="maximum">
            <double>0.100000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="solutionsLabel">
           <property name="text">
            <string>Cantidad de soluciones:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="modelParametersGroupBox">
     <property name="title">
      <string>Parámetros del modelo</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="bedsLabel">
        <property name="text">
         <string>Cantidad de capas:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="bedsSpinBox">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>252</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QTableWidget" name="bedsTableWidget">
        <property name="columnCount">
         <number>4</number>
        </property>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="processGroupBox">
     <property name="title">
      <string>Proceso</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QProgressBar" name="processProgressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="processPushButton">
        <property name="text">
         <string>Procesar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>defaultParametersRadioButton</tabstop>
  <tabstop>customParametersRadioButton</tabstop>
  <tabstop>temperatureDoubleSpinBox</tabstop>
  <tabstop>iterationsSpinBox</tabstop>
  <tabstop>movesPerTemperatureSpinBox</tabstop>
  <tabstop>solutionsSpinBox</tabstop>
  <tabstop>errorDoubleSpinBox</tabstop>
  <tabstop>bedsSpinBox</tabstop>
  <tabstop>bedsTableWidget</tabstop>
  <tabstop>processPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>defaultParametersRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>VFSAInversionDialog</receiver>
   <slot>selectedParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customParametersRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>VFSAInversionDialog</receiver>
   <slot>selectedParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VFSAInversionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VFSAInversionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectedParametersChanged()</slot>
 </slots>
</ui>
